### DISCUSSION TITLE
Use zygisk modul

### DISCUSSION THREAD

**Discussion by Momons18** [community/community#137580] [Programming Help]
### Select Topic Area

Question

### Body

Can someone explain me how to use zygisk module,already download it to .Zip but i don't know how to use it,thankyou

**Comment by Momons18** [comment]
![Screenshot_20240903-144236_Chrome](https://github.com/user-attachments/assets/f6c1afa9-fc18-482a-9115-44f1a7b3b094)
 This one is module what i want to use

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Momons18 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
